pallets/charity/src/tests.rs

```diff
@@ -168,6 +179,16 @@ fn imbalances_work() {
 		// Check that the correct event is emitted
 		let expected_event = TestEvent::charity(RawEvent::ImbalanceAbsorbed(5, new_pot_total));
 
+		// testing if the the event come in the correct order
+		assert_eq!(
+			System::events()[0],
+			EventRecord {
+				phase: Phase::Initialization,
+				event: TestEvent::charity(RawEvent::ImbalanceAbsorbed(5, new_pot_total)),
+				topics: vec![],
+			},
+		);
+
```

Comment:
This is perfect!

Reply:
Except that we don't really need the comment
```suggestion
		assert_eq!(
			System::events()[0],
			EventRecord {
				phase: Phase::Initialization,
				event: TestEvent::charity(RawEvent::ImbalanceAbsorbed(5, new_pot_total)),
				topics: vec![],
			},
		);

```

Reply:
ok, I will remove it.